### DISCUSSION TITLE
NumericUpDown add InnerLeftContent and InnerRightContent

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#14314] [Ideas]
Allow customizing `NumericUpDown` like `TextBox` does without recreating its template.

**Comment by workgroupengineering** [comment]
@timunie please give me if i can make PR.

**Reply by timunie** [reply]
I cannot answer this tbh. While I think it could be a good idea, also a Style for `Selector="NUD /template/ TextBox#PART_TextBox` can be used to style it without needing to override the template and the team may think this is enough 🤔

**Reply by workgroupengineering** [reply]
Thanks for the quick answer. I knew the possibility of using the Selector, but in my case it was not very usable. In the same view I have multiple `NumericUpDown` with different InnerLeftContents. I solved it by subcalsing `NumericUpDown`. But I think it is more useful for tuutic to have it by default. Ping me if you decide this would be a useful modification.

**Reply by rabbitism** [reply]
Actually you can use attached property as work around

**Reply by workgroupengineering** [reply]
Thanks, I hadn't thought of that

**Reply by timunie** [reply]
@workgroupengineering we have mixed feelings here. If you still think it's worth the effort, you can go ahead with a PR.

**Reply by workgroupengineering** [reply]
I'll do it next week.
